content/ri/using-redisinsight/api-get.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: Adding Databases via GET URL
+date: 2021-01-11 00:00:00 +0000
+weight: 16
+categories: ["RI"]
+path: api/
+nextStep:
+    Title: Memory Analysis
+    href: /docs/features/memory-analysis/
+---
+
+If you want to automate adding a Redis database without filling the [database form]({{< relref "/ri/using-redisinsight/add-instance.md" >}}), you might want to add database via GET URL.
+
+Below is the documentation for the URL required to add databases.
+
+## Add Redis database
+
+Used to add Redis databases to RedisInsight.
+
+**URL** : `/add/`
+
+**Method** : `GET`
+
+
+### Query Parameters
+
+These are the required query parameters for any type of database.
+
+| Parameter        | Type    | Required/Optional | Description                                                                  |
+|------------------|---------|-------------------|------------------------------------------------------------------------------|
+| `host`           | string  | required          | Hostname of your Redis database.                                             |
+| `port`           | number  | required          | Port of your Redis database.                                                 |
+| `name`           | string  | optional          | A nick name for the Redis database. Any string is valid                      |
+| `username`       | string  | optional          | Username of your Redis database.                                             |
+| `passsword`      | string  | optional          | Password of your Redis database.                                             |
+| `tls`            | boolean | optional          | `"true"` if your Redis database has TLS enabled.                             |
```

Comment:
Why is this an option if the note below says you need to use the database form if you add TLS certs?

Reply:
A Redis database can have TLS enabled without requiring TLS certificates to be added.
The most simple case is that the Redis database simply has TLS enabled.
Then there are other more advanced configurations that would require TLS certificates to be added (in case client authentication or a custom CA certificate is required).

<img width="719" alt="Screenshot 2021-02-05 at 1 42 29 PM" src="https://user-images.githubusercontent.com/25577002/107007503-9d8f7500-67b8-11eb-8e76-2b9b37c67372.png">

The `tls` parameter in the URL spec is only for the option marked `1`, and not the following options marked `2`. If those options need to be configured, the user would have to use the UI to add their Redis database.

Reply:
In the image above, the "Verify TLS Certificate" is supported with the `verifyServer` parameter.